views/appView.js

```diff
@@ -0,0 +1,36 @@
+var AppView = Backbone.View.extend({
+  initialize: function(model){
+    this.model = model;
+    //console.log("Here is my appModel");
+    //console.log(this.model);
+    this.listenTo(this.model.currentVideo, "change", this.render);
+    $("#search").on("click", this.search.bind(this));
+  },
+  events: {
+
+  },
+  search: function(){
+    var query = $("#search-term").val().trim().split(" ").join("+");
+    this.model.getVideos(query);
+  },
+  render: function(){
+    //check for child views, make them if they don't exist, updae them if they do
+    if(!this.mainVideoView){
```

Comment:
Ah, I see what you did here.  So you're doing too much work here.  Once you make the view and its listening for the model, then you shouldn't need to "re-set" the model or recreate the views like this.  Let me know if this doesn't make sense.  I know this is due to your not using the fetch function, but if you leverage the framework, this is greatly simplified.